Hudl.Mjolnir/Config/BulkheadsConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Hudl.Mjolnir.Config
+{
+    public class BulkheadsConfiguration
```

Comment:
Nit: use singular `Bulkhead` here instead of `Bulkheads`. Also don't forget to change the filename.